l1-contracts/deploy-scripts/PrepareZKChainRegistrationCalldata.s.sol

```diff
@@ -0,0 +1,286 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+// solhint-disable no-console
+
+import {Script, console2 as console} from "forge-std/Script.sol";
+import {stdToml} from "forge-std/StdToml.sol";
+
+import {IBridgehub} from "contracts/bridgehub/IBridgehub.sol";
+import {Bridgehub} from "contracts/bridgehub/Bridgehub.sol";
+import {L2ContractHelper} from "contracts/common/libraries/L2ContractHelper.sol";
+import {AddressAliasHelper} from "contracts/vendor/AddressAliasHelper.sol";
+import {L1SharedBridge} from "contracts/bridge/L1SharedBridge.sol";
+import {IGovernance} from "contracts/governance/IGovernance.sol";
+import {Utils} from "./Utils.sol";
+
+contract PrepareZKChainRegistrationCalldataScript is Script {
+    using stdToml for string;
+
+    address internal constant ADDRESS_ONE = 0x0000000000000000000000000000000000000001;
+
+    struct Config {
+        address chainAdmin;
+        address bridgehub;
+        address stateTransitionProxy;
+        uint256 chainId;
+        uint256 eraChainId;
+        uint256 bridgehubCreateNewChainSalt;
+        address baseToken;
+        bytes diamondCutData;
+        address governance;
```

Comment:
how does 'governance' differ from chain admin? could we fetch it from bridehub instead?

Reply:
governance here is our L1 governance contract, and yes it can be fetched as bridgehub.owner(), done that

Reply:
ok - so is there a reason to keep this field in the config?

Reply:
same as with bridgehub, it's fetched from the chain, just set here for convenience